website/source/docs/connect/ca/aws.html.md

```diff
@@ -0,0 +1,146 @@
+
+The set of configuration options is listed below. The
+first key is the value used in API calls while the second key (after the `/`)
+is used if configuring in an agent configuration file.
+
+  * `ExistingARN` / `existing_arn` (`string: <optional>`) - The Amazon Resource
+    Name (ARN) of an existing private CA in your ACM account. If specified,
+    Consul will attempt to use the existing CA to issue certificates. Currently
+    this **must be a root CA** in the primary datacenter although in the future
+    we intend to support subordinate CAs. In a secondary datacenter, it must be
+    a subordinate CA who's root is the same as the root used in the primary
+    datacenter - if it's not Consul will create a new one from the Primary
+    datacenter's root.
+
+      The default behavior if this is not specified is for Consul to create a
+      new Root CA in the primary datacenter and a subordinate CA in each
+      secondary DC that it will automatically get signed by the primary's root.
+
+## Limitations
+
+ACM Private CA has several
+[limits](https://docs.aws.amazon.com/acm-pca/latest/userguide/PcaLimits.html)
+that restrict how fast certificates can be issued. This may impact how quickly
+large clusters can rotate all issued certificates.
+
+At the current time the ACM Private CA provider for Connect has some additional
+limitations described below.
+
+### Unable to Cross-sign Other CAs
+
+It's currently not possible to cross-sign other CA provider's root certificates
+during a migration. This is due to a ACM Private CA not having a mechanism to
+blindly cross-sign another root certificate without a CSR being generated. Both
+Consul's built in CA and Vault can do this and the current workflow for managing
+CAs relies on it.
+
+In the future it should be possible to remove this limitation by changing the
+way Consul manages CA certificates significantly.
+
+For now, the limitation means that once ACM Private CA is configured as the CA
+provider, it is not possible to reconfigure a different CA provider, or rotate
+the root CA key without potentially observing some transient connection
+failures. See the section on [Forced rotation without
+cross-signing](/docs/connect/ca.html#forced-rotation-without-cross-signing) for
+more details.
+
+### Primary DC Must be a Root CA
+
+Currently, if an existing ACM Private CA is used, the primary DC must use a Root
+CA directly to issue certificates. We expect to remove this limitation in the
+future.
+
+## Cost Example
+
+To help estimate costs, an example is provided below of the resources that would
+be used.
+
+~> This is intended to illustrate the behavior of the CA for cost planning
+purposes. Please refer to the [pricing for ACM Private
+CA](https://aws.amazon.com/certificate-manager/pricing/) for actual cost
+information.
+
+Assume the following Consul datacenters exist and all will use ACM Private CA as
```

Comment:
```suggestion
Assume the following Consul datacenters exist and all agents will use ACM Private CA as
```

Reply:
It's really the datacenter rather than the agents that use ACM since CA provider is configured per DC. Does it read better as:

> Assume the following Consul datacenters exist and are configured to use ACM Private CA...